Update 20.treatments (#365)
greenelab/covid19-review: content/20.treatments.md

As recently discussed [@doi:10.1093/jac/dkaa114], because the 90% effective concentration (EC~90~) of CQ in Vero E6 cells (6.90 μM) can be achieved in and tolerated by rheumatoid arthritis patients, it may also be possible to achieve the effective concentration in COVID-19 patients.
Additionally, HCQ was found to be effective in HIV [@doi:10/cq2hx9] and chronic Hepatitis C [@doi:10.1002/jmv.24575] patients.
Therefore, initially it was thought that CQ/HCQ would be effective against SARS-COV-2.
There are few reasons why HCQ was used more commonly than CQ.

> The hypotheses were that HCQ could alleviate the development of COVID-19 through mitigation of CD154 in T cells, could have the same effect as an antiviral like CQ pre- and post-infection, is cheaper, and has less side effects than CQ

Reviewer: I think you wanted this sentence struck before (#340 or #338)-- did you want to workshop it in this version?

Author: yes

Reviewer: @soumitagh OK! I would suggest a little change because my understanding is that HCQ is just the synthetic version of CQ:
```suggestion
CQ and HCQ have both been found to inhibit the expression of CD154 in T-cells and to reduce toll-like receptor signaling that leads to the production of pro-inflammatory cytokines [@doi:10.1038/s41584-020-0372-x], but HCQ offers the advantages of being cheaper and having fewer side effects than CQ.
```

https://www.ncbi.nlm.nih.gov/pubmed/27911847
-->
The lack of benefits indicated by large studies of COVID-19 treatment with HCQ (e.g., [@doi:10/ggwzsb]) is important to recognize clinically because these medications are also associated with dangerous side effects like prolonged QT interval [@doi:10.1155/2016/4626279].

There is now evidence of increased mortality among older men receiving HCQ, and administration of HCQ and HCQ+AZ did not decrease the use of mechanical ventilation in the admitted patients [@doi:10.1101/2020.04.16.20065920]. 
Although [@doi:10/ggwzsb] was initially thought to show that healthy patients had an increased risk of _de novo_ ventricular arrhythmia, this study has now been retracted. 
An important concern not disclosed in the original publication was that whether the patients administered with HCQ/CQ had similar viral load or similar symptoms as the controls.

> There have still not been any studies that investigated HCQ/CQ efficacy using randomized controls.

Reviewer 2: I'm not following all the HCQ literature, but is this new sentence still accurate now that https://www.nejm.org/doi/full/10.1056/NEJMoa2016638 is out?

> We conducted a randomized, double-blind, placebo-controlled trial across the United States and parts of Canada testing hydroxychloroquine as postexposure prophylaxis. We enrolled adults who had household or occupational exposure to someone with confirmed Covid-19 at a distance of less than 6 ft for more than 10 minutes while wearing neither a face mask nor an eye shield (high-risk exposure) or while wearing a face mask but no eye shield (moderate-risk exposure). Within 4 days after exposure, we randomly assigned participants to receive either placebo or hydroxychloroquine (800 mg once, followed by 600 mg in 6 to 8 hours, then 600 mg daily for 4 additional days). The primary outcome was the incidence of either laboratory-confirmed Covid-19 or illness compatible with Covid-19 within 14 days.

Reviewer: @agitter I was actually just about to submit a new PR to address this and add the info from #366 !

https://www.ncbi.nlm.nih.gov/pubmed/27911847
-->
The lack of benefits indicated by large studies of COVID-19 treatment with HCQ (e.g., [@doi:10/ggwzsb]) is important to recognize clinically because these medications are also associated with dangerous side effects like prolonged QT interval [@doi:10.1155/2016/4626279].

> However, [@doi:10/ggwzsb] is currently retracted after several concerns being raised.

Reviewer: ```suggestion
```
I think this ended up on the wrong line

There is now evidence of increased mortality among older men receiving HCQ, and administration of HCQ and HCQ+AZ did not decrease the use of mechanical ventilation in the admitted patients [@doi:10.1101/2020.04.16.20065920]. 

> Even otherwise healthy patients had an increased risk of _de novo_ ventricular arrhythmia [@doi:10/ggwzsb].

Reviewer: ```suggestion
```

> However, [@doi:10/ggwzsb] is retracted. 

Reviewer: ```suggestion
Although [@doi:10/ggwzsb] was initially thought to show that healthy patients had an increased risk of _de novo_ ventricular arrhythmia, this study has now been retracted. 
```

> An importent concern not disclosed in the original publication was that whether the patients administered with HCQ/CQ 

Reviewer: ```suggestion
```

> had similar viral load or similar symptoms to the disease as the controls.

Reviewer: ```suggestion
An important concern not disclosed in the original publication was that whether the patients administered with HCQ/CQ had similar viral load or similar symptoms as the controls.
```
Is this in reference to the retracted study? If so, I think it is a little confusing because (my understanding is) that is not why it was retracted. Perhaps you would want to reiterate why it was retracted and then say that this is also a significant concern?

> All the same, the study was not randomized controlled.

Reviewer: ```suggestion
There have still not been any studies that investigated HCQ/CQ efficacy using randomized controls.
```
